modules/navier_stokes/tests/ins/jacobian_test/jacobian_supg_test.i

```diff
@@ -0,0 +1,146 @@
+[GlobalParams]
+  gravity = '0 0 0'
+  integrate_p_by_parts = false
+[]
+
+[Mesh]
+  type = GeneratedMesh
+  dim = 3
+  xmax = 1.1
+  ymax = 1.1
+  zmax = 1.1
+  nx = 1
+  ny = 1
+  nz = 1
+  elem_type = HEX8
+[]
+
+[Variables]
+  [./vel_x]
```

Comment:
This test uses equal-order interpolation for velocity and pressure, so it is not LBB stable. Although it might work for the specific grid used in this test, you would potentially encounter issues if you try to refine the mesh, etc. I'd rather not have input files with unstable formulations in the module, because people create new simulations and tests by copying existing ones...